train/DirectoryStatistics.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace DirectoryStatistics
+{
+    class day5
+    {
+        public static long length;
+        public static double size;
+        //save the length and size of the directory
+        public static void Main(string[] args)
+        {
+            /*
+            #if DEBUG
+                        args = new[] { "-d /Users/admin/Desktop/dailyreport -p *.txt" };
+            #endif
+            */
+            string filePath;
+            string pattern;
+            Stopwatch watch = new Stopwatch();
+            watch.Start();
+            if (args.Length > 0 && args[0] == "-d" && args[2] == "-p")
```

Comment:
For this part, you may consider below point:
args.Length>0 won't guarantee how many items are in array "args", so that it may throw index out of range exception in args[2], args[1] and args[3].